### ISSUE TITLE
Visual Studio 2022 doesn't work

### ISSUE THREAD

**Issue by MrMarvel** [madskristensen/CloseAllTabs#18]
### Installed product versions
- Visual Studio:  2022 Enterprise
- This extension: 1.4.35

### Description
It doesn't delete files that I want.

### Steps to recreate
1. Open project
2. Enable deleting .vs, bin, obj folders.
3. Create git repository with default autocreated by Visual Studio .gitignore
4. Run project
5. Close Visual Studio

### Current behavior
It doesn't delete vs, bin and obj folders.

### Expected behavior
It have to delete vs, bin and obj folders

**Comment by zburns** [comment]
Agreed - does not work or install with Microsoft Visual Studio Professional 2022 (ARM 64-bit) - Current Version 17.4.5
And no dice with 17.5.1

**Comment by damian-666** [comment]
i bet this is a quick fix.. ill sponsor cause am running out of space all the time
